modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/PrimaryRecipients.tsx

```diff
@@ -49,12 +49,13 @@ export function PrimaryRecipient({
 		const newRecipients: EmailNotificationRecipients[] = [];
 
 		if (items.length) {
-			const [itemsGroup] = items as MultiSelectItem[];
-
-			itemsGroup.children.forEach((child) => {
-				if (child.checked) {
-					newRecipients.push({['roleName']: child.value});
-				}
+			const itemsGroup = items as MultiSelectItem[];
+			itemsGroup.map((itemGroup) => {
+				itemGroup.children.forEach((child) => {
+					if (child.checked) {
+						newRecipients.push({['roleName']: child.value});
+					}
+				});
```

Comment:
Hey @igor-franca, this line `52` is unnecessary, you can use `items` directly or rename it to `itemsGroup` in the function parameter.

Also, if you are not going to return anything, you don't need to use the `map` function, use `forEach` instead